books/free-programming-books.md

```diff
@@ -161,6 +162,7 @@
 * [Modelica](#modelica)
 * [MySQL](#mysql)
 * [Neo4J](#neo4j)
+* [.NET Core / .NET](#net-core)
```

Comment:
Is the name ".NET Core / .NET" ok?

I named it like that because  the latest version in .NET can be called either .NET Core 5 or simply .NET 5

Later in Future, .NET Core part can be removed when a new version of .NET is released

Reply:
The audience for the headings is folks looking for books on one or the other. There will also be people like me who may not be sure of the difference. If you're unsure, ask a colleague. Just make sure the heading links work.

Reply:
I'm sure about this, then let's go with current name - ".NET Core / .NET" to make more sense

Explanation from [Microsoft Docs](https://docs.microsoft.com/en-us/dotnet/core/dotnet-five) for reference: 
> .NET 5.0 is the next major release of .NET Core following 3.1. We named this new release .NET 5.0 instead of .NET Core 4.0 for two reasons:

> We skipped version numbers 4.x to avoid confusion with .NET Framework 4.x.
We dropped "Core" from the name to emphasize that this is the main implementation of .NET going forward. .NET 5.0 supports more types of apps and more platforms than .NET Core or .NET Framework.
ASP.NET Core 5.0 is based on .NET 5.0 but retains the name "Core" to avoid confusing it with ASP.NET MVC 5. Likewise, Entity Framework Core 5.0 retains the name "Core" to avoid confusing it with Entity Framework 5 and 6.

Reply:
it's ready to merge, if you feel they aren't more changes to be made

Reply:
ok. will double check the link

Reply:
checked and good